pages/common/oh-my-posh.md

```diff
@@ -0,0 +1,36 @@
+# oh-my-posh
+
+> Cross platform prompt engine for any shell.
+> More information: <https://ohmyposh.dev/docs>.
```

Comment:
Is there a comprehensive man page for this tool that lists all its capabilities?